packages/backend/src/clients/EmailClient/templates/imageNotification.html

```diff
@@ -1,490 +1,528 @@
+  <title>Simple Transactional Email</title>
+  <style>
+      /* -------------------------------------
+      GLOBAL RESETS
+  ------------------------------------- */
+
+      /*All the styling goes here*/
+
+      img {
+          border: none;
+          -ms-interpolation-mode: bicubic;
+          max-width: 100%;
+      }
+
+      body {
+          background-color: #e5e8eb;
+          font-family: 'Roboto', sans-serif;
+          -webkit-font-smoothing: antialiased;
+          font-size: 14px;
+          line-height: 1.4;
+          margin: 0;
+          padding: 0;
+          -ms-text-size-adjust: 100%;
+          -webkit-text-size-adjust: 100%;
+      }
+
+      table {
+          border-collapse: separate;
+          mso-table-lspace: 0pt;
+          mso-table-rspace: 0pt;
+          width: 100%;
+      }
+
+      table td {
+          font-family: 'Roboto', sans-serif;
+          font-size: 14px;
+          vertical-align: top;
+      }
+
+      /* -------------------------------------
+      BODY & CONTAINER
+  ------------------------------------- */
+
+      .body {
+          background-color: #e5e8eb;
+          width: 100%;
+      }
+
+      /* Set a max-width, and make it display as block so it will automatically stretch to that width, but will also shrink down on a phone or something */
+      .container {
+          display: block;
+          margin: 0 auto !important;
+          /* makes it centered */
+          max-width: 660px;
+          padding: 50px;
+          width: 660px;
+          text-align: center;
+      }
+
+      /* This should also be a block element, so that it will fill 100% of the .container */
+      .content {
+          box-sizing: border-box;
+          display: block;
+          margin: 0 auto;
+          max-width: 660px;
+          padding: 10px;
+      }
+
+      /* -------------------------------------
+      HEADER, FOOTER, MAIN
+  ------------------------------------- */
+
+      .logo {
+          margin: 10px 0;
+          width: 116px;
+          height: 25px;
+      }
+
+      .miniature-logo {
+          width: 39px;
+          height: 39px;
+          margin-right: 15px;
+      }
+
+      .social {
+          width: 20px;
+          height: 20px;
+          margin: 0 10px 0 10px;
+      }
+
+      .main {
+          border-radius: 3px;
+          width: 100%;
+      }
+
+      .wrapper {
+          box-sizing: border-box;
+          padding: 64px;
+          padding-bottom: 20px;
+
+      }
+
+      .header {
+          display: flex;
+          flex-direction: row;
+          margin-bottom: 5px;
+      }
+
+      .title {
+          font-family: 'Roboto', sans-serif;
+          font-size: 20px;
+          line-height: 28px;
+          font-weight: 500;
+          color: #171818;
+          margin: 0;
+      }
+
+      .content-block {
+          padding-bottom: 10px;
+          padding-top: 10px;
+      }
+
+      .footer {
+          clear: both;
```

Comment:
this is a bad practice, why was this necessary?

Reply:
don't know, was already there! I can remove it